### DISCUSSION TITLE
Options to Remove Preferences

### DISCUSSION THREAD

**Discussion by maheshrajrp** [eclipse-theia/theia#9899] [General]
Hi All,
I've been trying to remove selected preferences. Consider this example, Let's say I don't find any use to the Features tab in the Preference (as shown in the pic below), and I would like it to be removed. If any functions are dependent on the preferences then they can use the default values that are assigned. It would be great to have this functionality,   a function that would allow us to remove the selected Preference Class and so, the user wont be able to see the functionality that aren't deployed. As of now I tried unbinding the class, this results in the application not starting up. 

![image](https://user-images.githubusercontent.com/80001447/129245157-b14c3b4f-254f-4b9e-b16b-9b1aabde1507.png)

**Comment by vince-fugnitto** [comment]
@maheshrajrp the screenshot you have is of an older application, the preferences are grouped much more differently now.

The use-case can be achieved by removing the preferences (by rebinding the schema), customizing the `preferences-view` (to suit your needs), or by changes proposed in https://github.com/eclipse-theia/theia/pull/8626 which allow you to `hide` preferences.
